index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.10/css/all.css" integrity="sha384-+d0P83n9kaQMCwj8F4RJB66tzIwOKmrdb46+porD/OvrJ+37WqIM7UoBtwHO6Nlg" crossorigin="anonymous">
+  <link href="style.css" rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Kaushan+Script' rel='stylesheet' type='text/css'>
+  <meta charset="utf-8">
+  <title>Ask Joey</title>
+</head>
+
+<body>
+  <div class="header">
+    <div class="col-md-8 offset-md-2">
+      <h1 class="pageTitle">AskJoey.com</h1>
+    </div>
+  </div>
+
+  <!-- Main Page -->
+  <div class="container col-md-8 offset-md-2 mainPage">
+    <div class="row">
+      <div class="postHeader col-md-8 text-center">
```

Comment:
It's convention to not use "dash delimited" syntax for class names in CSS as opposed to camel case - so `post-header`